packages/slice-machine/src/utils/tracking/trackFieldAdded.ts

```diff
@@ -0,0 +1,28 @@
+import {
+  Group,
+  NestableWidget,
+  UID,
+} from "@prismicio/types-internal/lib/customtypes";
+
+import { telemetry } from "@/apiClient";
+import { SlicePrimaryFieldSM } from "@/legacy/lib/models/common/Slice";
+
+import { getContentTypeForTracking } from "./getContentTypeForTracking";
+
+export function trackFieldAdded(
+  id: string,
+  field: SlicePrimaryFieldSM | NestableWidget | UID | Group,
+) {
+  void telemetry.track({
+    event: "field:added",
+    id,
+    name: field.config?.label ?? "",
+    type: field.type,
+    isInAGroup: false,
```

Comment:
⚠️ Hum, here you're just removing a tracking feature, it should be true when calling from inside a group.

Reply:
yeah, I wasn't sure about that, in all 3 places that I replace with new method it was hard coded to false, thought there is some reason for that 😅 will fix that then

Reply:
i guess it was never relly implemented? lol

curious why product never complained about this, then

Reply:
It was not:

Old code:
```
void telemetry.track({
      event: "field:added",
      id,
      name: label,
      type: newField.type,
      isInAGroup: true,
      contentType: getContentTypeForTracking(window.location.pathname),
      ...(newField.type === "Link" && {
        allowText: newField.config?.allowText,
      }),
    });
```

Reply:
fixed